packages/create-svelte/scripts/update-template-repo-contents.js

```diff
@@ -21,3 +21,18 @@ await create(repo, {
 	prettier: true,
 	playwright: false
 });
+
+// Remove the Sverdle from the template because it doesn't work within Stackblitz (cookies not set)
+fs.rmSync(path.join(repo, 'src', 'routes', 'sverdle'), { force: true, recursive: true });
+
+const header = fs.readFileSync(path.join(repo, 'src', 'routes', 'Header.svelte'), 'utf-8');
+fs.writeFileSync(
+	path.join(repo, 'svelte.config.js'),
```

Comment:
svelte.config.js?

Might wanna add some comments here, it's not immediately clear what's happening here, and for the regexes as well.